server/db/models/EventType.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const schema = new Schema({
+    user_id: {
+        // Link to appointments using user_id?
+        type: Number,
```

Comment:
This should also probably be an ObjectId but maybe we can leave that until we know what the user setup will be